paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ComposeFormDataCollectionFragment.kt

```diff
@@ -58,6 +58,8 @@ internal class ComposeFormDataCollectionFragment : Fragment() {
             ViewGroup.LayoutParams.MATCH_PARENT
         )
 
+        setPadding(0, 18, 0, 0)
```

Comment:
What is this padding for? Do we use dimens.xml, or something similar?

Reply:
I don't think we have been consistent, this adds some extra padding before the first element in the compose view.